setup.py

```diff
@@ -99,7 +99,7 @@ def read_version():
         'sqlalchemy>=2.0.0,<2.1.0',
         'requests',
         'clickhouse-driver>=0.1.2',
-        'asynch>=0.2.2,<=0.2.4',
+        'asynch>=0.2.2,<0.4.0',
```

Comment:
Hello.

The version v0.2.4 and v0.2.5 are advisable to be excluded: the v0.2.4 might have (if I am not mistaken) and the v0.2.5 has issues with connection management handling logic (an [example](https://github.com/long2ice/asynch/pull/129)).

The v0.3.0 seems to be OK except some bugs that are currently under the progress (consider having a look at [this PR](https://github.com/long2ice/asynch/pull/138)) and a minor improvement [here](https://github.com/long2ice/asynch/pull/136).

Reply:
This may do:
```suggestion
        'asynch>=0.2.2,!=0.2.4,!=0.2.5<0.4.0',
```
https://packaging.python.org/en/latest/specifications/version-specifiers/#id5